models/marts/fact_bigquery_job_statistics.sql

```diff
@@ -39,10 +39,8 @@ with
             , (total_bytes_billed/(1048576)) as total_billed_mb
             , (total_bytes_billed/(1073741824)) as total_billed_gib
             , (total_bytes_billed/(1099511627776)) as total_billed_tib
-            , (total_bytes_billed/(1099511627776) * {{ var('price_per_tib', '1') }}) as query_cost_usd
-            , (total_bytes_billed/(1099511627776) * (
```

Comment:
Didnt understand the change in the number here

Reply:
Previously, i was mutiplying the price per tib processed by the dollar cotation in BRL. But this is wrong. To know the price in BRL, i have to multiply the total bytes billed by 40.167405531. As this cost depends on the region, i added it as a variable

Reply:
![image](https://github.com/user-attachments/assets/7041ba3b-602e-42c6-9c51-7891ee7f1f1f)